openstack_controller/tests/test_unit_nova.py

```diff
@@ -1268,6 +1268,7 @@ def test_hypervisor_uptime_exception(aggregator, check, dd_run_check, mock_http_
         for call in mock_http_get.call_args_list:
             args, _ = call
             args_list += list(args)
+        print(args_list)
```

Comment:
nit: I'm guessing that this is a debugging statement.

Reply:
Good catch!